ui/views.py

```diff
@@ -117,42 +116,28 @@ class VideoDetail(TemplateView):
 
     def get_context_data(self, video_key, **kwargs):  # pylint: disable=arguments-differ
         context = super().get_context_data(**kwargs)
-        default_settings = json.loads(default_js_settings(self.request))
         video = get_object_or_404(Video, key=video_key)
-        context['video'] = video
-        context['form'] = forms.VideoForm(instance=video)
-        try:
-            videofile = video.videofile_set.get(encoding=EncodingNames.HLS)
-            default_settings['videofile'] = videofile.cloudfront_url
-        except VideoFile.DoesNotExist:
-            videofile = None
-            default_settings['videofile'] = None
-
-        context['videofile'] = videofile
-        context["js_settings_json"] = json.dumps(default_settings)
+        context["js_settings_json"] = json.dumps({
+            **default_js_settings(self.request),
+            'videoKey': video.key.hex,
+        })
```

Comment:
Right, I'll see about passing this information into the react components to show this information